.gitignore

```diff
@@ -85,6 +86,7 @@ deps/icu*.zip
 deps/icu*.tgz
 deps/icu-tmp
 ./node_modules
+node_modules/
```

Comment:
Why is this nececery? Or could it replace the one above it?

Reply:
Update: `./gitignore` was added so that node_modules folder from other places (like npm/tools) isn't ignored in commit https://github.com/nodejs/node/commit/25e8ea17e1f03cd40cc5a6c9c1987bae58313ff5